src/server_manager/web_app/ui_components/app-root.ts

```diff
@@ -391,6 +395,11 @@ export class AppRoot extends polymerElementWithLocalize {
 
           <!-- Links section -->
           <paper-listbox>
+            <a class="documentation-link" 
+              href="https://www.reddit.com/r/outlinevpn/wiki/index/">
+                <span>[[localize('outline-documentation')]]</span>
+                <img src="images/open_in_newlink.svg" />
```

Comment:
Why not use `<iron-icon>` as used elsewhere (e.g. a few lines below) for consistency?

Reply:
The main reason I suggested it is so you can use the default icon set so we don't need to add our own copied `open_in_newlink.svg` image file. That icon (`open-in-new`) is already loaded as part of the default icon set that is imported in this file.